src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -178,14 +197,14 @@ public void testAbove() throws Exception {
     public void testInFlow() throws Exception {
         final DefaultConfiguration checkConfig =
             createModuleConfig(ImportOrderCheck.class);
+        checkConfig.addAttribute("groups", "java.awt, javax.swing, java.io");
         checkConfig.addAttribute("option", "inflow");
         final String[] expected = {
             "6: " + getCheckMessage(MSG_ORDERING, "java.awt.Dialog"),
             "11: " + getCheckMessage(MSG_ORDERING,
                      "javax.swing.WindowConstants.DO_NOTHING_ON_CLOSE"),
             "12: " + getCheckMessage(MSG_ORDERING, "javax.swing.WindowConstants.*"),
             "13: " + getCheckMessage(MSG_ORDERING, "javax.swing.JTable"),
-            "15: " + getCheckMessage(MSG_ORDERING, "java.io.File.createTempFile"),
```

Comment:
this change is not expected.
It was valid violation before your change, it should stay like this.

Reply:
Because I add Attribute groups,and it was not vaild violation.
Done.